content/2022/01/minfraud-now-supports-8-digit-iins.mdx

```diff
@@ -0,0 +1,83 @@
+---
+title: "minFraud services now support 8 digit IINs"
+date: "2022-01-31"
+featuredImage: minfraud.svg
+categories:
+  - "featured-entry-3"
```

Comment:
@christophermluna Although it isn't currently a functional feature, we should still delete `featured-entry-3` from the previous post that has it.

Reply:
my understanding is that the implementation was designed not to need to remove that category from previous posts. i was told that the implementation was designed such that we always take the most recent blog post with <featured-#> category. to me, that makes more sense. is there another need to delete? we'll probably also clean this up in #181029999

Reply:
Perhaps I misunderstood. Was this implementation for the old or new blog? Or both?

Reply:
We're gonna change the implementation.  Instead of using `featured-entry-x`, we'll be using [isFeatured: true](https://github.com/maxmind/blog-site/blame/adding-category-pages/content/2021/07/how-accurate-is-ip-geolocation.mdx#L13)